### ISSUE TITLE
mongo array support

### ISSUE THREAD

**Issue by jney** [stripe/mosql#100]
Maybe i missed this from documentation, Is it possible to extract data from an array ?

ie :
```yaml
mongodb:
  collection:
    :columns:
    - latitude:
      :source:  location[0]
      :type: FLOAT
    - longitude:
      :source:  location[1]
      :type: FLOAT
```

**Comment by allpratik** [comment]
Same Issue. Will the above stuff work?

**Comment by arnists** [comment]
Did you test this?

**Comment by jney** [comment]
@arnists excuse me for the delay.
i tested it and i have got a `[null]` instead of the content

**Comment by JosephZeng1121** [comment]
hi, any thoughts on this issue. Our mongo structure is full of array. It's really hard to parse jason array in tableau

**Comment by allpratik** [comment]
@JosephZeng1121 AFA in my case, we ditched Tableau and used Pentaho through KTR files. Pentaho is not that visually appealing as of Tableau, but it is versatile and helped us to get our work done.

**Comment by bitliner** [comment]
@JosephZeng1121 a way could be to update the code and add the support for arrays.

I had a look at it, and I think what should be changed is after [this line](https://github.com/stripe/mosql/blob/master/lib/mosql/streamer.rb#L144)
and at `schema.transform(...)` method.

Can this help?
